numpy/core/src/multiarray/scalartypes.c.src

```diff
@@ -1333,74 +1333,46 @@ gentype_transpose_get(PyObject *self)
 static PyGetSetDef gentype_getsets[] = {
     {"ndim",
         (getter)gentype_ndim_get,
-        (setter) 0,
-        "number of array dimensions",
-        NULL},
+        (setter) 0, NULL, NULL},
     {"flags",
         (getter)gentype_flags_get,
-        (setter)0,
-        "integer value of flags",
-        NULL},
+        (setter)0, NULL, NULL},
     {"shape",
         (getter)gentype_shape_get,
-        (setter)0,
-        "tuple of array dimensions",
-        NULL},
+        (setter)0, NULL, NULL},
     {"strides",
         (getter)gentype_shape_get,
-        (setter) 0,
-        "tuple of bytes steps in each dimension",
-        NULL},
+        (setter) 0, NULL, NULL},
     {"data",
         (getter)gentype_data_get,
-        (setter) 0,
-        "pointer to start of data",
-        NULL},
+        (setter) 0, NULL, NULL},
     {"itemsize",
         (getter)gentype_itemsize_get,
-        (setter)0,
-        "length of one element in bytes",
-        NULL},
+        (setter)0, NULL, NULL},
     {"size",
         (getter)gentype_size_get,
-        (setter)0,
-        "number of elements in the gentype",
-        NULL},
+        (setter)0, NULL, NULL},
     {"nbytes",
         (getter)gentype_itemsize_get,
-        (setter)0,
-        "length of item in bytes",
-        NULL},
+        (setter)0, NULL, NULL},
     {"base",
         (getter)gentype_base_get,
-        (setter)0,
-        "base object",
-        NULL},
+        (setter)0, NULL, NULL},
     {"dtype",
         (getter)gentype_typedescr_get,
-        NULL,
-        "get array data-descriptor",
-        NULL},
+        NULL, NULL, NULL},
     {"real",
         (getter)gentype_real_get,
-        (setter)0,
-        "real part of scalar",
-        NULL},
+        (setter)0, NULL, NULL},
     {"imag",
         (getter)gentype_imag_get,
-        (setter)0,
-        "imaginary part of scalar",
-        NULL},
+        (setter)0, NULL, NULL},
     {"flat",
         (getter)gentype_flat_get,
-        (setter)0,
-        "a 1-d view of scalar",
```

Comment:
Some of these may actually have been better docstrings in a sense. OTOH, refering the ndarray attribute is nice.